### PR TITLE
MultivaluedSection.values() swift4

### PR DESCRIPTION
**Pull request by bmichotte**
This is the swift 4 port of #1215

### CODE REVIEW

File: Source/Core/Section.swift [hunk 1]
```diff
@@ -470,4 +470,13 @@ open class MultivaluedSection: Section {
         }
         self <<< addRow
     }
+
+    /**
+     Method used to get all the values of the section.
+
+     - returns: An Array mapping the row values. [value]
+     */
+    public func values() -> [Any?] {
```
Comment (mtnbarreto) [inline Source/Core/Section.swift]: why does it public?

File: Source/Core/Form.swift [hunk 2]
```diff
@@ -360,6 +352,22 @@ extension Form {
         }
         kvoWrapper.sections.insert(section, at: formIndex == NSNotFound ? 0 : formIndex + 1 )
     }
+
+    func getValues(for rows: [BaseRow]) -> [String: Any?] {
```
Comment (mtnbarreto) [inline Source/Core/Form.swift]: could it be private?

File: Source/Core/Section.swift (hunk 1 above)
Comment (mtnbarreto) [reply Source/Core/Section.swift]: I would prefer to remove this method

Comment (bmichotte) [reply Source/Core/Section.swift]: This way, you can call 
```
let section = form.sectionBy(tag: "sectionTag")
section.values()
```

File: Source/Core/Form.swift (hunk 2 above)
Comment (bmichotte) [reply Source/Core/Form.swift]: indeed

**Pull request merged**